### PR TITLE
Fixes #29086 - Configure and fix Style/TrailingCommaIn*Literal

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

**Comment by lzap** [comment]
Is it possible to make this a non-fatal check? Like a warning. While I understand it's good, I also hate to hear from Rubocop when I miss it.

File: modules/dhcp_common/dhcp_common.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@ module Proxy::DHCP
     :install_server_name   => {:code => 11, :kind => "String"    }, # mediahost
     :install_path          => {:code => 12, :kind => "String"    }, # /vol/solgi_5.10/sol10_hw0910
     :sysid_server_path     => {:code => 13, :kind => "String"    }, # 192.168.216.241:/vol/jumpstart/sysidcfg/sysidcfg_primary
-    :jumpstart_server_path => {:code => 14, :kind => "String"    }  # 192.168.216.241:/vol/jumpstart
+    :jumpstart_server_path => {:code => 14, :kind => "String"    },  # 192.168.216.241:/vol/jumpstart
```
Comment (ekohl) [inline modules/dhcp_common/dhcp_common.rb]: Perhaps we should keep the comments aligned
```suggestion
    :jumpstart_server_path => {:code => 14, :kind => "String"    }, # 192.168.216.241:/vol/jumpstart
```

**Review by ekohl — APPROVED** [review]

**Comment by mmoll** [comment]
> Is it possible to make this a non-fatal check?

AFIACT no.

File: modules/dhcp_common/dhcp_common.rb (hunk 1 above)
Comment (mmoll) [reply modules/dhcp_common/dhcp_common.rb]: Fixed.

**Pull request merged**